.ci.yaml

```diff
@@ -259,41 +259,53 @@ targets:
     bringup: true
     recipe: engine_v2/engine_v2
     timeout: 60
+    dimensions:
+      os: "Linux"
     properties:
       release_build: "true"
       config_name: linux_fuchsia
 
   - name: Linux linux_arm_host_engine
     recipe: engine_v2/engine_v2
     timeout: 60
+    dimensions:
+      os: "Linux"
```

Comment:
No, this property is being used to detect the drone names, if we use ubuntu it will translate to `Ubuntu Production Drone`

Reply:
Done_dimensions are not determining the `Platform` of the drone builder. It is the platform of the orchestrator builder instead.

Reply:
Please take a look to: https://cs.opensource.google/flutter/recipes/+/main:recipe_modules/shard_util_v2/api.py;l=271

Reply:
My bad based on a local checkout which is not up to date.

`Linux` should be fine then. Could we keep the `Mac` ones with the version to enforce same config? We do have consistent all Mac-12 in our fleet as of now, but just in case we have different versions. This makes it consistent with all other `os` dimension.